### PR TITLE
RC1 Changelog.

### PR DESCRIPTION
**Pull request by vpetrovykh**
This is the first iteration. I assume we'll add more details potentially about the way we process environment and default settings, etc. This would affect CLI and bindings.

### CODE REVIEW

File: docs/changelog/1_0_rc1.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 RC 1 "Ran".
+
+
+EdgeQL
+======
+
+* Drop ``FINAL`` from the syntax (:eql:gh:`#2607`).
```
Comment (1st1) [inline docs/changelog/1_0_rc1.rst:12]: Have we actually dropped the syntax? @msullivan

Comment (vpetrovykh) [reply docs/changelog/1_0_rc1.rst:12]: From what I understand we no longer support it for most cases:
```
db> create final scalar type foo extending str;
error: UnsupportedFeatureError: FINAL is not supported
  ┌─ query:1:1
  │
1 │ create final scalar type foo extending str;
  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ error
```
But I guess we have a way of allowing it in dumps and migrations: #2886

So as a language feature it's not longer part of any actual construct and is deprecated, but not outright an error for some legacy cases. Which I think makes it fair to say that we've "dropped" it as we don't actually want anyone to use it.

**Comment by vpetrovykh** [comment]
I've added the "Bindings" section with a mention of new protocol version. The release versions for Python and JS bindings are just guesses based on the last version that I see for those bindings. The Go driver version appears to be a recent release, so I used the version we already released (around Aug 26).

**Comment by vpetrovykh** [comment]
I've also reorganized a bit by splitting off schema-related stuff from the EdgeQL section into the Migrations section.

File: docs/changelog/1_0_rc1.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  sets expensive due to explicit elimination of duplicates.
+  Furthermore, non-distinct scalar sets are actually *desirable* in
+  many queries, especially where analytics and tuples are involved.
+  Finally, we should enforce consistent multiplicity rules on computed
+  properties, and static inference of multiplicity on scalars is weak
+  and will effectively force users to pollute queries with pointless
+  and expensive :eql:op:`DISTINCT`.
+
+  Here we change EdgeDB behavior to allow multiplicity greater than
+  one in ``multi`` properties.
```
Comment (1st1) [inline docs/changelog/1_0_rc1.rst:49]:
```suggestion
  Here we change EdgeDB behavior to allow multiplicity *greater than
  one* in ``multi`` properties.
```

**Review by 1st1 — APPROVED** [review]

**Pull request merged**